File: src/main/java/ar/edu/unlam/tallerweb1/domain/Services.java
package ar.edu.unlam.tallerweb1.domain;

public class Services {

    public static FriendsGroupService getFriendsGroupService() {
        return new FriendsGroupService(
                Repositories.Companion.getUserRepository(),
                Repositories.Companion.getFriendsGroupRepository(),
                Repositories.Companion.getSharedExpensesRepository(),
                Repositories.Companion.getPaymentRepository());
    }
}

File: src/main/java/ar/edu/unlam/tallerweb1/domain/Services.kt
package ar.edu.unlam.tallerweb1.domain

import ar.edu.unlam.tallerweb1.domain.Repositories.getFriendsGroupRepository
import ar.edu.unlam.tallerweb1.domain.Repositories.getPaymentRepository
import ar.edu.unlam.tallerweb1.domain.Repositories.getSharedExpensesRepository
import ar.edu.unlam.tallerweb1.domain.Repositories.getUserRepository

object Services {
    fun getFriendsGroupService() = FriendsGroupServiceImpl(
        getUserRepository(),
        getFriendsGroupRepository(),
        getSharedExpensesRepository(),
        getPaymentRepository()
    )
}